backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/contract/contractPeriod/extendModularContractPeriod.feature

```diff
@@ -0,0 +1,105 @@
+Feature: Extend Modular Contract Period
+
+  Background:
+    Given infrastructure and metasfresh are running
+    And the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And metasfresh has date and time 2022-03-01T13:30:13+01:00[Europe/Berlin]
+    And set sys config boolean value true for sys config SKIP_WP_PROCESSOR_FOR_AUTOMATION
+
+    And load C_Calendar from metasfresh:
+      | C_Calendar_ID.Identifier | Name                  |
+      | harvesting_calendar      | Buchführungs-Kalender |
+
+    And load C_Year from metasfresh:
+      | C_Year_ID.Identifier | FiscalYear | OPT.C_Calendar_ID.Identifier |
```

Comment:
The C_Year.C_Calendar_ID column is mandatory in DB. Please remove the `OPT.` prefix from it.